README.md

```diff
@@ -147,49 +147,37 @@ Rev 2 PCBs come with ARM 10-pin SWD headers and can be programmed via ST-Link. A
 ### ST-Link Software Installation
 
 ### Ubuntu
```

Comment:
Discussed offline, but just add a comment to say that ubuntu 18 is not supported atm